doc/man7/fips_module.pod

```diff
@@ -470,6 +470,17 @@ L<OSSL_PROVIDER_get0_name(3)>.
 
 =head1 NOTES
 
+Some released versions of OpenSSL do not include a validated
+FIPS provider.  To determine which versions have undergone
+the validation process, please refer to the
+L<OpenSSL Downloads page|https://www.openssl.org/source/>.  If you
+require FIPS-approved functionality, it is essential to build your FIPS
+provider using one of the validated versions listed there.  Normally,
+it is possible to utilize a FIPS provider constructed from one of the
+validated versions alongside F<libcrypto> and F<libssl> compiled from any
+release within the same major release series.  This flexibility enables
+you to address bug fixes and CVEs that fall outside the FIPS boundary.
+
```

Comment:
Would it make sense to have (part of) this information also in ´NEWS.md`?

Reply:
IMO this does not belong to NEWS.md

Reply:
A note saying 3.0.8 achieve FIPS validation might be worth including in NEWS.md, however it's always going to lag the release (i.e. it would be in 3.0.10 changes now).